pkg/wf/extractor.go

```diff
@@ -308,6 +308,47 @@ func extractFile(path string, p parser, config *ExtractorConfig) ([][]byte, erro
 	return p.output(data, config)
 }
 
+func packFile(path string, p parser, config *ExtractorConfig) error {
```

Comment:
I'm considering creating a separate `Packer` for packing functionalities instead of bundling them together within `Extractor`.